### ISSUE TITLE
Investigate if weapon buff spells add too much damage

### ISSUE THREAD

**Issue by mlenser** [mlenser/kryx-rpg-issues#864]
Weapon buff spells like flaming weapon add ~3 DPR in tier 1, ~6 DPR in tier 2, **~12 DPR** in tier 3 and tier 4, and **~17 DPR** in tier 5.

This is from a half caster. A full caster would add significantly more. RAW only increases 3 times whereas mine increase 5x. I think increasing at 1, 3, and 5 mana would be much more balances (still as unbalanced as above when cast by a full caster though...).

Further investigation needed.

**Comment by itamarcu** [comment]
In tier 3, the half caster spends 2 mana to add ~12 DPR every turn, for the duration of 10 minutes concentration (usually 1 combat, rarely 2).  Alternatively, that 2 mana could have been used to cast a spell like Harm (~30 damage).  The half caster is sacrificing immediate damage for future damage (enemies can survive longer), risking losing concentration on the spell, and becomes unable to cast another concentration spell.  Additionally, enemies who see that one party member has magically burning swords can do the smart thing and keep their distance from that gish or use CC to prevent attacks.

Based on the sheet, the spells are balanced, so I think to call them unbalanced you'd need to explain what wrong assumptions are made by the sheet.

By the way, if anything I think the 1-mana version of the spell is *weak*, in tier 1, because it adds only 1d6 damage per turn instead of 2d6 (since the character only makes one attack per turn).  Of course, this becomes fair starting in T2, but it still seems a bit sad that the lowest level characters shouldn't cast that spell.

**Comment by mlenser** [comment]
Round delay and damage in subsequent rounds is already calculated. However those are less of a factor for spells like these that can be cast outside of combat. There isn't a tradeoff when cast optimally. Concentration is the only risk.

> Based on the sheet, the spells are balanced, so I think to call them unbalanced you'd need to explain what wrong assumptions are made by the sheet.

The sheet isn't perfect always. Using another model like I did with DPR can expose shortcomings. That said, it should be the main basis for balancing numeric spells where possible.
Buffs are generally not easy to model with the sheet, but we can try.

One thing that I noticed is that I treat self buffs the same value as a spell cast on an enemy. A self buff will last as long as the duration (minus concentration) whereas the spell cast on an enemy will last as long as that enemy is alive (~4 rounds). So changing the self buffs to be longer exposes that 1d6 was too much and it should be 1d4.

The same logic would apply to Enlarge.


> By the way, if anything I think the 1-mana version of the spell is weak, in tier 1, because it adds only 1d6 damage per turn instead of 2d6 (since the character only makes one attack per turn). Of course, this becomes fair starting in T2

It is indeed weak because it would become much stronger with 2 actions. Enlarge is the same.

The only solution would be to change it to 2 mana so it can't be used at 1 mana at all.
